easybuild/easyconfigs/q/QUDA/QUDA-1.1.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,40 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'QUDA'
+version = '1.1.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/lattice/quda'
+description = """QUDA is a library for performing calculations in lattice QCD
+ on graphics processing units (GPUs), leveraging NVIDIA's CUDA platform."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'lattice'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['b4f635c993275010780ea09d8e593e0713a6ca1af1db6cc86c64518714fcc745']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Python', '3.10.4'),
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', True),
+    ('UCX-CUDA', '1.12.1', versionsuffix),
+    ('Eigen', '3.4.0'),
+    ('NVSHMEM', '2.6.0', versionsuffix),
```

Comment:
@Wahid612 `NVSHEM` 2.7.0 was added in #16789, so can we bump this dependency to `2.7.0`?

If so, please also remove the easyconfig for NVSHEM 2.6.0 you've included in this PR (see https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches)